be/src/util/mem_info.cpp

```diff
@@ -122,8 +122,6 @@ bool MemInfo::process_minor_gc() {
     MonotonicStopWatch watch;
     watch.start();
     int64_t freed_mem = 0;
-    std::string vm_rss_str = PerfCounters::get_vm_rss_str();
-    std::string mem_available_str = MemInfo::sys_mem_available_str();
```

Comment:
Record the system memory status before GC for printing logs. Otherwise, "be mem < mem_limit" or `system available memory > water mark` may appear in the log during GC, causing confusion to users

Reply:
already modified, thanks